packages/toast-ui.grid/cypress/integration/export.spec.ts

```diff
@@ -0,0 +1,310 @@
+import GridEvent from '@/event/gridEvent';
+import { cls } from '@/helper/dom';
+import { RowKey } from '@t/store/data';
+import { SinonStub } from 'cypress/types/sinon';
+import { invokeFilter, setSelectionUsingMouse } from '../helper/util';
+
+before(() => {
+  cy.visit('/dist');
+});
+
+function showContextMenu(rowKey: RowKey, columnName: string) {
+  cy.getCell(rowKey, columnName).trigger('contextmenu');
+}
+
+function getMenuItemByText(text: string) {
+  return cy.contains(`.${cls('context-menu')} .menu-item`, text);
+}
+
+function formatToType(format: string) {
+  if (format === 'csv') {
+    return 'CSV';
+  }
+  return 'Excel';
```

Comment:
```suggestion
  return format === 'csv' ? 'CSV' : 'Excel';
```
이렇게 줄이는건 어떤가요?
format이 `'csv'`랑 `'excel'`이라면 `toUpperCase`로 리턴해도 괜찮을 것 같아요.

Reply:
아래를 보니 `'xlsx'`라서 `toUpperCase`는 안되겠네요.